content/doc/book/using/remote-api.adoc

```diff
@@ -1,37 +1,281 @@
 ---
-layout: redirect
-redirect_url: https://wiki.jenkins.io/display/JENKINS/Remote+access+API
+layout: section
+title: Remote access API
 ---
 
-////
+= Remote access API
 
----
-layout: section
----
-:notitle:
-:description:
-:author:
-:email: jenkinsci-docs@googlegroups.com
-:sectanchors:
-:toc:
-:hide-uri-scheme:
+Jenkins provides machine-consumable remote access API to its
+functionalities. Currently it comes in three flavors:
+
+. XML
+. JSON with JSONP support
+. Python
+
+Remote access API is offered in a REST-like style. That is, there is no
+single entry point for all features, and instead they are available
+under the `+".../api/"+` URL where `+"..."+` portion is the data that it
+acts on.
+
+For example, if your Jenkins installation sits at http://ci.jruby.org/,
+visiting http://ci.jruby.org/api/ will show just the top-level API
+features available – primarily a listing of the configured jobs for this
+Jenkins instance. +
+Or if you want to access information about a particular build, e.g.
+http://ci.jruby.org/job/jruby-base/lastSuccessfulBuild/, then go to
+http://ci.jruby.org/job/jruby-base/lastSuccessfulBuild/api/ and you'll
+see the list of functionalities for that build.
+
+The work on this front is ongoing, so if you find missing features,
+please file an issue.
+
+[[RemoteaccessAPI-Whatcanyoudowithit]]
+== What can you do with it?
+
+Remote API can be used to do things like these:
+
+. retrieve information from Jenkins for programmatic consumption.
+. trigger a new build
+. create/copy jobs
+
+[[RemoteaccessAPI-Submittingjobs]]
+== Submitting jobs
+
+*Jobs without parameters*
+
+You merely need to perform an HTTP POST on
+`+JENKINS_URL/job/JOBNAME/build+`.
+
+*Jobs with parameters*
+
+Also see
+https://wiki.jenkins.io/display/JENKINS/Parameterized+Build[Parameterized
+Build].
+
+Simple example - sending "String Parameters":
+
+[source,syntaxhighlighter-pre]
+----
+curl -X POST JENKINS_URL/job/JOB_NAME/build \
+  --user USER:TOKEN \
+  --data-urlencode json='{"parameter": [{"name":"id", "value":"123"}, {"name":"verbosity", "value":"high"}]}'
+----
```

Comment:
Build log says 

```

[2020-05-15T08:31:47.456Z] While processing file /home/jenkins/workspace/Infra_jenkins.io_PR-3262/content/doc/book/using/remote-api.adoc

[2020-05-15T08:31:47.456Z] An error occurred: Invalid id given: syntaxhighlighter-pre
```

Maybe better to just use another simple syntax

```
\```
curl -X POST JENKINS_URL/job/JOB_NAME/build \
  --user USER:TOKEN \
  --data-urlencode json='{"parameter": [{"name":"id", "value":"123"}, {"name":"verbosity", "value":"high"}]}'
\```
```

Please remove the "\" when applying the suggestion

Reply:
Just to be clear on this @oleg-nenashev, `--data-urlencode json='{"parameter": [{"name":"id", "value":"123"}, {"name":"verbosity", "value":"high"}]}'` becomes `--data-urlencode json='{parameter: [{name:id, value:123}, {name:verbosity, value:high}]}'` and if that's the case should I do the same in all other occurrences in the document ?